GerritDownloader/src/main/java/com/holmsted/gerrit/CommandLineParser.java

```diff
@@ -153,6 +161,18 @@ public int getCommitLimit() {
         return limit;
     }
 
+    public String getAfterDate() {
```

Comment:
I kept Date as String because it used like this by gerrit query command. I added a coverter to catch parse exception and display usage if format is bad.